app/src/main/java/net/gsantner/opoc/util/GsContextUtils.java

```diff
@@ -982,9 +983,16 @@ public String getMimeType(final Context context, String uri) {
             mimeType = GsFileUtils.getMimeType(new File(uri));
         }
 
+        // Detect extensions which open in this app at plain text
+        // Done only if all other tests fail
+        if (GsTextUtils.isNullOrEmpty(mimeType) && new AppSettings().init(context).isExtOpenWithThisApp(ext)) {
```

Comment:
Markor AppSettings shouldn't be in GsContextUtils, I will check what I can do here.

Reply:
I guess we can create an `isPlainText` file function somewhere in markor. But I think we have too many mime type / isTextFile functions rn, each with different semantics.

Reply:
How did this check get here? I recall the ext check to be explicitley there where we do also the other extension-to-type association. At converters...and this one being in PlaintextConverter.

Reply:
I added it to make sure snippets and templates showed all possible files.

We could probably just move it out to the actual snippet and template logic for now

Reply:
Is FormatRegistry.isFileSupported()  what you look for?

Reply:
I think that will do. Pushed the change. 

I also think we should work on unifying the various mime type / file type / isTextFile functions. There are way too many now.